### ISSUE TITLE
Getting `loom` to work from the Windows command line

### ISSUE THREAD

**Issue by JobLeonard** [linnarsson-lab/loom-viewer#139]
It would be great if we could avoid requiring Windows users to compile everything from source.

http://www.voidspace.org.uk/python/articles/command_line.shtml#path

**Comment by JobLeonard** [comment]
This should remove the need for anaconda in the path, but still requires compilation:

```batch
set anaconda_path="conda info --root"
%anaconda_path%\python.exe .\python\loom_viewer\loom %*
```

**Comment by JobLeonard** [comment]
So the problem _may_ be that Anaconda does not include `pylauncher` on windows:
https://stackoverflow.com/questions/30794342/python-windows-where-is-the-python-launcher

It can be installed separately:
https://bitbucket.org/vinay.sajip/pylauncher

This _may_ remove the need for a `loom.bat` file, and also make loom-viewer pip-installable. Will test tonight (when I get home and have access to my Windows machine)

**Comment by JobLeonard** [comment]
Ok, so after looking ta this some more I think I finally have figured out two overlapping problems are, and how to fix it by doing something different altogether:

1. The Anaconda installation of Python does not include [the pylauncher](https://bitbucket.org/vinay.sajip/pylauncher/overview)
1. As part of the installation of Anaconda, it adds itself to the path. Assuming it is the main Python installation, `pip install ...` saves installed python packages in `C:\Anaconda3\Scripts\`, including our `loom` file.
1. However, since `loom` has no extension, even with `pylauncher` it doesn't work in Windows. If I rename it to `loom.py` and then type `loom.py` in the windows command line after installing `pylauncher`, everything works.

Plan to fix:

- [] use [py2exe](http://www.py2exe.org/) to build a Windows executable `loom.exe` as part of the build process
- [] include this executable version of loom as part of the distribution

Since it will be installed to the `Scripts` folder, it should now be executable from anywhere, and without typing the full extension.

This should remove the need for compiling the loom viewer from source for windows users (and all the baggage that it brings with it)

**Comment by JobLeonard** [comment]
Nopenopenope, py2exe isn't the answer either. That just creates a completely standalone exe file, which would require including all SciPi dependencies. Even assuming this is possible to begin with, that's ridiculously huge.

So finally, I figured "maybe I can trick `setup.py` to include a `loom.bat` file to make everything work?", and then I discuvered this in the SetupTools documentation:

> Feature highlights:
>
> - ...
> - ***Automatically generate wrapper scripts or Windows (console and GUI) .exe files for any number of “main” functions in your project.*** (Note: this is not a py2exe replacement; the .exe files rely on the local Python installation.)

https://setuptools.readthedocs.io/en/latest/setuptools.html

Ok... should have looked here first I guess.
